timecache/timeCache.go

```diff
@@ -1,60 +1,58 @@
 package timecache
 
 import (
-	"sync"
 	"time"
 
 	"github.com/ElrondNetwork/elrond-go-storage/common"
-	"github.com/ElrondNetwork/elrond-go-storage/types"
 )
 
-type span struct {
-	timestamp time.Time
-	span      time.Duration
+// RequestedItemsHandler can determine if a certain key has or not been requested
+type RequestedItemsHandler interface {
```

Comment:
this interface is only used here to check the implementation. We can drop the check & interface definition